src/components/contacts/contactsPage.jsx

```diff
@@ -128,36 +173,61 @@ var contactsPage = React.createClass({
 							</th>
+							<th>
+								<h5>Reminders</h5>
+							</th>
 						</table>
 					</div>
 					<div className="panel-body table-responsive scroll">
 						<table className="table table-full">
 							<tbody>
 
-						  		{ this.state.contacts.map(function (contact, i) {
-							        return <tr>
-							            		<td key={i + "first"}>
-							            			<a onClick={contactHandler.bind(null, contact)}>
-							            				<p>{contact.company_name}</p>
-							            			</a>
-							            		</td>
-							            		<td key={i + "second"}>
-							            			<a onClick={contactHandler.bind(null, contact)}>
-							            				<p>{contact.category}</p>
-							            			</a>
-							            		</td>
-							            		<td key={i + "third"}>
-							            			<a onClick={contactHandler.bind(null, contact)}>
-							            				<p>{contact.city}</p>
-							            			</a>
-							            		</td>
-							            		<td key={i + "fourth"}>
-							            			<a onClick={contactHandler.bind(null, contact)}>
-							            				<p>{contact.postcode}</p>
-							            			</a>
-							            		</td>
-											</tr>
-							    })}
+						  		{this.state.contacts !== 0
+							  		? this.state.contacts.map(function (contact, i) {
+
+							  			var cont = contact.sort(function (a, b) {
+							  				return b.week - a.week
+										})
+										
+								        return <tr key={i}>
+								            		<td>
+								            			<a onClick={contactHandler.bind(null, cont[0])}>
+								            				<p>{cont[0].company_name}</p>
+								            			</a>
+								            		</td>
+								            		<td>
+								            			<a onClick={contactHandler.bind(null, cont[0])}>
+								            				<p>{cont[0].category}</p>
+								            			</a>
+								            		</td>
+								            		<td>
+								            			<a onClick={contactHandler.bind(null, cont[0])}>
+								            				<p>{cont[0].city}</p>
+								            			</a>
+								            		</td>
+								            		<td>
+								            			<a onClick={contactHandler.bind(null, cont[0])}>
+								            				<p>{cont[0].postcode}</p>
+								            			</a>
+								            		</td>
+								            		<td>
+								            			<a>
+															<p className='align-center'>
+																<svg onClick= {reminderHandler.bind(null, contact)} className="bell" viewBox="0 0 88 72" >
+																	{(cont[0].week && cont[0].week === 1)
+												            			? <path className="present" d="M71.5,38.184h-3.291l-6.213-21.879c-1.367-4.816-5.951-8.693-10.904-9.514C50.91,3.02,47.812,0,43.996,0c-3.812,0-6.91,3.018-7.092,6.787c-4.957,0.822-9.539,4.697-10.908,9.514l-6.211,21.883h-3.289l-4.498,15.85h19.251H36h15.994h3.963h20.041L71.5,38.184z M40.975,6.611C41.229,5.143,42.455,4,43.996,4c1.543,0,2.77,1.143,3.025,2.615L40.975,6.611z"/>
```

Comment:
I would consider making the svg into its own component as to not disrupt the readability of this code. Just a suggestion, but certainly not overly important.